src/test/java/core/basesyntax/service/impl/WriterServiceImplTest.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.WriterService;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class WriterServiceImplTest {
```

Comment:
add test, when you try to write into wrong path

Reply:
Can't get RuntimeException like in my "catch" block in WriterServiceImpl. I added test that expecting nullpointerexc.class;